src/pages/iou/request/step/IOURequestStepWaypoint.js

```diff
@@ -100,6 +113,65 @@ function IOURequestStepWaypoint({
         }
     }, [parsedWaypointIndex, waypointCount]);
 
+    // Construct the rectangular boundary based on user location, waypoints and direction coordinates
+    const locationBias = useMemo(() => {
+        // If there are no filled wayPoints and if user's current location cannot be retrieved,
+        // it is futile to arrive at a biased location. Let's return
+        if (filledWaypointCount === 0 && _.isEmpty(userLocation)) {
+            return null;
+        }
+
```

Comment:
@parasharrajat 
If there are no waypoints and also user's current location cannot be determined, do you think we should consider the default coordinates in the app i.e. [-122.4021, 37.7911] as set [here](https://github.com/Expensify/App/blob/58ae7fe8fabde71059d0599a48a35bff062f246c/src/CONST.ts#L2940)? 
or returning `null` as it is done currently is good enough?